fabric-lifecycle-events-v1/src/main/java/net/fabricmc/fabric/api/client/event/lifecycle/v1/ClientEntityEvents.java

```diff
@@ -57,12 +57,12 @@ public ClientEntityEvents() {
 	/**
 	 * Called when an Entity is about to be unloaded from a ClientWorld.
 	 *
-	 * <p>When this event is called, the entity is still present in the world.
+	 * <p>This event is called before the entity is removed from the world.
```

Comment:
Perhaps this should be "This event is called before the entity is unloaded from the world" to be consistent?

Reply:
Ummm Well.. That is good!!